src/lib/bootloader/config_dialog.rb

```diff
@@ -19,11 +19,17 @@ class ConfigDialog
 
     def run
       guarded_run
-    rescue ::Bootloader::BrokenConfiguration => e
-      ret = Yast::Report.AnyQuestion(_("Broken Configuration"),
+    rescue ::Bootloader::BrokenConfiguration, ::Bootloader::UnsupportedOption => e
+      msg = if e.is_a?(::Bootloader::BrokenConfiguration)
         # TRANSLATORS: %s stands for readon why yast cannot process it
         _("YaST cannot process current bootloader configuration (%s). " \
-          "Propose new configuration from scratch?") % e.reason,
+          "Propose new configuration from scratch?") % e.reason
+      else
+        e.message
+      end
+
+      ret = Yast::Report.AnyQuestion(_("Unsupported Configuration"),
+        msg,
         _("Propose"),
         _("Quit"),
```

Comment:
"Quit" means exiting which does not make much sense during installation. The message says "repropose or edit manually" so the appropriate label could be `Edit` or something like that.

Reply:
IT cannot happen during installation. It is runtime only where user edit configuration with something else then YaST2.

Reply:
And edit manually means edit by something other than yast2 bootloader.